docs/_sources/ciservices.rst.txt

```diff
@@ -8,11 +8,18 @@ CircleCI (Linux)
 Circle CI is a container-based CI service that conda-forge uses to build
 linux packages.
 
+
+Travis CI (OSX)
+------------------------------
+Travis CI is used to build packages for OSX. After merging a staged-recipes pull request, it might be necessary to
+force sync your repositories in Travis CI to see the reload and cancel buttons. To do this please visit "https://travis-ci.org/profile/<github_user_name>" and click "Sync accounts".
+
+
```

Comment:
Can you move this up `CircleCI (Linux)` or all the way down. This ended up in the middle :smile:  (` Enabling Circle on your Fork` is a subpart of `CircleCI (Linux)`).